src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -7073,6 +7073,44 @@ namespace VariableSpeedCoils {
         return MinOAT;
     }
 
+    Real64 GetVSCoilMinOATCompressorUsingIndex(int const CoilIndex, // index to cooling coil
+                                               bool &ErrorsFound    // set to true if problem
+    )
+    {
+
+        // FUNCTION INFORMATION:
+        //       AUTHOR         R. Raustad
+        //       DATE WRITTEN   August 2019
+
+        // PURPOSE OF THIS FUNCTION:
+        // This function looks up the the min oat for the cooling coil compressor and returns it.  If
+        // incorrect coil type or name is given, ErrorsFound is returned as true and value is returned
+        // as negative.
+
+        // Return value
+        Real64 MinOAT; // returned min oa temperature of matched coil
+
+        // Obtains and Allocates WatertoAirHP related parameters from input file
+        if ( GetCoilsInputFlag ) { // First time subroutine has been entered
```

Comment:
No need for the padding around the if condition contents, that's the old format.  I'm a bit surprised that the CI isn't warning on this.